tests/unittests/CoreText/CTFontTests.mm

```diff
@@ -147,6 +147,7 @@ virtual void TearDown() {
     EXPECT_NE(0, CTFontGetUnderlineThickness(font));
     EXPECT_NE(0, CTFontGetCapHeight(font));
     EXPECT_NE(0, CTFontGetXHeight(font));
+    EXPECT_EQ(0, CTFontGetLeading(font));
```

Comment:
You should also update CTCatalog with the methods that are working now that were stubbed out before